TFT/src/User/API/parseACK.c

```diff
@@ -1078,7 +1078,9 @@ void parseACK(void)
         infoMachineSettings.autoReportTemp = ack_value();
 
         if (infoMachineSettings.autoReportTemp)
-          storeCmd("M155 ");
+        {
```

Comment:
it should be better to not use {} in an "if" with only one instruction (as done in all the rest of the file)

Reply:
Fixed.